app/features/workout/WorkoutScreen.js

```diff
@@ -4,10 +4,10 @@ import { connect } from 'react-redux';
 import * as SetsSelectors from 'app/redux/selectors/SetsSelectors';
 import * as DateUtils from 'app/utility/transforms/DateUtils';
 import * as RepDataMap from 'app/utility/transforms/RepDataMap';
-import * as SetTimeCalculator from 'app/utility/transforms/SetTimeCalculator';
```

Comment:
Add SetTimeCalculator back in for this entire file.

Rebase fucked up somehow. Need SetTimeCalculator or else shit will break.

Reply:
Done

Reply:
Done